docs/usage/exporting_models.md

```diff
@@ -162,21 +187,31 @@ class FooBarModel(BaseModel):
 m = FooBarModel(foo=datetime(2032, 6, 1, 12, 13, 14), bar={'whatever': 123})
 print(m.model_dump_json())
 #> {"foo":"2032-06-01T12:13:14","bar":{"whatever":123}}
+print(m.model_dump_json(indent=2))
+"""
+{
+  "foo": "2032-06-01T12:13:14",
+  "bar": {
+    "whatever": 123
+  }
+}
+"""
 ```
 
-### `json_encoders`
+### Custom serializer
 
-Serialisation can be customised on a model using the `json_encoders` config property; the keys should be types (or names of types for forward references), and
-the values should be functions which serialise that type (see the example below):
+Serialisation can be customised on a field and model using `field_serializer` and `model_serializer` decorators. (see the example below):
```

Comment:
Should we add a note mentioning that previously you'd solve this with the `json_encoders` setting?